tests/Makefile

```diff
@@ -74,30 +74,38 @@ gen_file_dep = |
 endif
 
 .PHONY: generated_files
+GENERATED_DATA_FILES :=
```

Comment:
https://github.com/Mbed-TLS/mbedtls/pull/8559 contains some of the build script commits from this pull request, somewhat adapted:

* parts of 300568f169140efadad6752ed20470fbe0410128 + 0ff6d8bcc85325be5f24061cea532f200a0558c3 → 927cabea4f6b512ab0945233e6a14cb04b0e6a37
* 6ca7cd7be0c34f28f050053dfebf1dcce0324b4f → fe5aed98698d764221713d40ff5db46acc10b9ad